packages/flutter/lib/src/widgets/widget_span.dart

```diff
@@ -85,6 +87,37 @@ class WidgetSpan extends PlaceholderSpan {
         ),
       );
 
+  /// Helper function for extracting [WidgetSpan]s in preorder, from the given
+  /// [InlineSpan] as a list of widgets.
+  ///
+  /// The extracted widgets will be annotated with necessary semantics
+  /// information. This function is used by [EditableText] and [RichText] so
+  /// calling it directly is rarely necessary.
```

Comment:
This should document the `textScaleFactor` param.

Also, for my own understanding: I thought we are getting rid of textScaleFactor?

Reply:
That's the next step. If it's ok I think I'll just replace `textScaleFactor` with `textScaler` without deprecating in the migration PR given nobody but the framework widgets are calling this method.